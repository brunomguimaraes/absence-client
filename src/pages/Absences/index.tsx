
import React from 'react';
import AbsenceTable from '../../components/AbsenceTable';

const Absences = () => {
    return (
        <div id="page-absences">
          <AbsenceTable />
        </div>
    );
};

export default Absences;
