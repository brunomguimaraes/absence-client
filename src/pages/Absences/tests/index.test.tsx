import React from 'react';
import { shallow } from 'enzyme';

import Absences from '..';

describe('Absences', () => {
  it('renders the component', () => {
    const component = shallow(<Absences />);
    expect(component).toMatchSnapshot();
  });
});
