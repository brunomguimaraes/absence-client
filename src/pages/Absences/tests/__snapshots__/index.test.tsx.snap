// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Absences renders the component 1`] = `ShallowWrapper {}`;
